presto-tests/src/test/java/com/facebook/presto/memory/TestMemoryManager.java

```diff
@@ -474,7 +474,8 @@ public void testClusterPoolsMultiCoordinator()
 
             ClusterMemoryPoolInfo generalPool = memoryManager.getClusterInfo(GENERAL_POOL);
             assertNotNull(generalPool);
-
+            //Adding sleep for QueryTracker to register queries
+            MILLISECONDS.sleep(100);
```

Comment:
adding sleep is going to be non-deterministic I think.
It might succeed locally but fail again on jenkins/ci

Reply:
Modify the behavior of the test and removed the sleep.